corehq/apps/toggle_ui/templates/toggle/edit_flag.html

```diff
@@ -24,21 +24,39 @@
         $('.hq-help').hqHelp();
     });
     $(function(){
+        var PAD_CHAR = '&nbsp;'
         function ToggleView() {
             var self = this;
             self.items = ko.observableArray();
 
-            self.init = function (item_list) {
+            self.init = function (config) {
+                self.padded_ns = {};
+                var max_ns_len = Math.max.apply(Math, _.map(config.namespaces, function (ns) { return ns.length }));
+                _(config.namespaces).each(function (namespace) {
+                    var diff = max_ns_len - namespace.length,
+                        pad = new Array(diff + 1).join(PAD_CHAR);
+                    self.padded_ns[namespace] = namespace + pad;
+                });
+
+                self.init_items(config.items);
+            };
+
+            self.init_items = function (items) {
                 self.items.removeAll();
-                _(item_list).each(function (item) {
+                _(items).each(function (item) {
+                    var fields = item.split(':'),
+                        namespace = fields.length > 1 ? fields[0] : 'user',
+                        value = fields.length > 1 ? fields[1] : fields[0];
                     self.items.push({
-                        value: ko.observable(item)
+                        namespace: ko.observable(self.padded_ns[namespace]),
```

Comment:
I'm not super great at CSS, but could you have used `min-width` or something similar to do that?